scalatest/src/main/scala/org/scalatest/tools/Runner.scala

```diff
@@ -1447,7 +1448,7 @@ object Runner {
         clazz.newInstance.asInstanceOf[Suite]
       else {
         val suiteClazz = wrapWithAnnotation.value
-        val constructorList = suiteClazz.getDeclaredConstructors()
+        val constructorList : Array[Constructor[_]] = suiteClazz.getDeclaredConstructors().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.